bindings/python/src/session.cpp

```diff
@@ -413,6 +413,39 @@ namespace
         s.set_alert_notify(std::bind(&alert_notify, cb));
     }
 
+#ifdef TORRENT_WINDOWS
+    void alert_handle_notify(HANDLE const fd)
+    {
+        std::uint8_t dummy = 0;
+		::send(fd, &dummy, 1, 0);
+    }
+#endif
+
+    void alert_fd_notify(int const fd)
+    {
+        std::uint8_t dummy = 0;
+        while (::write(fd, &dummy, 1) < 0 && errno == EINTR);
+    }
+
+    void set_alert_fd(lt::session& s, int fd)
```

Comment:
Don't we want the arg to be larger than `int`? I thought `SOCKET`s are pointers.